src/pages/contact-us.astro

```diff
@@ -0,0 +1,115 @@
+---
+import Layout from "../layouts/Layout.astro";
+import { Content as FaqContent } from '../content/faq.md';
+import "../content/markdown-styles.css";
+
+const pageTitle = "Contact Us - UBC iGEM";
+---
+
+<Layout title={pageTitle}>
+  <style>
+    h1 {
+      font-size: 4.5em;
+      font-weight: 900;
+      text-decoration-color: #042a2b;
+    }
+    h2 {
+      font-size: 2em;
+      font-weight: 400;
+      margin-top: 0.5em;
+      margin-bottom: 0.5em;
+      text-decoration-color: #042a2b;
+    }
+    p {
+      font-size: 1.3em;
+      text-decoration-color: #042a2b;
+    }
+
+    .page-main-heading {
+      margin-bottom: 0.75em;
+    }
+
+    .faq-heading {
+      text-decoration-color: #042a2c;
+      margin-top: 0.5em;
+      margin-bottom: 0.25em;
+      font-weight: 700;
+    }
+
+    .contact-section {
+      background-color: #afdbbf;
+      padding-bottom: 5%;
+      padding-inline: 10%;
+      text-align: left;
+      border-top: solid 2.5px black;
+      border-bottom: solid 2.5px black;
+      text-decoration-color: #042a2a;
+      outline: black;
+      max-width: 100%;
+    }
+
+    .contact-row {
+      display: flex;
+      align-items: center;
+    }
+
+    .contact-row img {
+      width: 48px;
+      height: 48px;
+      flex-shrink: 0;
+      margin-right: 12px;
+    }
+
+    .contact-row a {
+      text-decoration: none;
+      color: inherit;
+    }
+
+    .contact-handle {
+      text-decoration-color: #383f57;
+    }
+
+    .contact-handle-email {
+      text-decoration: underline;
```

Comment:
you don't need this underline, it's just something figma added in (sorry, i know it looks confusing)